README.md

```diff
@@ -17,52 +20,71 @@ Zeppelin is a moderation bot for Discord, designed with large servers and reliab
 - Starboard
 - And more!
 
-See https://zeppelin.gg/ for more details.
+See <https://zeppelin.gg/> for more details.
 
 ## Development
-These instructions are intended for bot development only.
 
-👉 **No support is offered for self-hosting the bot!** 👈
+These instructions are intended for bot development only, they are not recommended for self-hosting.
 
-### Running the bot
-1. `cd backend`
-2. `npm ci`
-3. Make a copy of `bot.env.example` called `bot.env`, fill in the values
-4. Run the desired start script:
-    * `npm run build` followed by `npm run start-bot-dev` to run the bot in a **development** environment
-    * `npm run build` followed by `npm run start-bot-prod` to run the bot in a **production** environment
-    * `npm run watch` to watch files and run the **bot and api both** in a **development** environment
-      with automatic restart on file changes
-5. When testing, make sure you have your test server in the `allowed_guilds` table or the guild's config won't be loaded at all
-
-### Running the API server
-1. `cd backend`
-2. `npm ci`
-3. Make a copy of `api.env.example` called `api.env`, fill in the values
-4. Run the desired start script:
-    * `npm run build` followed by `npm run start-api-dev` to run the api in a **development** environment
-    * `npm run build` followed by `npm run start-api-prod` to run the api in a **production** environment
-    * `npm run watch` to watch files and run the **bot and api both** in a **development** environment
-      with automatic restart on file changes
+👉 **No support is guaranteed for self-hosting the bot!** 👈
+
+### Running the backend
+
+1. Go into the backend directory: `cd backend`
+
+2. Install dependencies: `npm ci`
+
+3. Make a copy of `bot.env.example` and `api.env.example` (removing the `.example` suffix), fill in the values.
+  There are defaults for your convenience, feel free to replace these.
+
+4. Setup the database schema, use `npm run migrate-dev`.
+
+5. Run `npm run build` followed by the desired start script:
+    - **Recommended** is to use `npm run watch`, this starts both the bot and api server and automatically restarts on save.
```

Comment:
`npm run build` is not required when using `npm run watch`. Perhaps the main text could be changed to something like "Run the bot and/or API server:", with `npm run build` and `npm run watch` offered as alternatives?